tests/bdd/features/volume/topology/pool-affinity.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Volume Topology
+  The volume topology feature is for selection of particular pools to schedule a replica on.
+  All the volume topology key and values should have an exact match with pool labels, although pools
```

Comment:
If pools can have extra labels then it's not an "exact match".

Reply:
Made the changes, Thanks for pointing out. Updated the test case as will